app/components/qualifications_table_component.html.erb

```diff
@@ -1,19 +1,18 @@
 <% if qualifications.any? %>
-  <div class="govuk-grid-column-two-thirds">
```

Comment:
I’ve removed this grid container, and thus reduced the indentation of each line.